engine/src/main/java/org/destinationsol/SolApplication.java

```diff
@@ -173,8 +173,14 @@ public void create() {
         nuiManager = new NUIManager(this, context, commonDrawer, options);
     }
 
+    @SuppressWarnings("checkstyle:ParameterAssignment")
     @Override
     public void resize(int newWidth, int newHeight) {
+        //To prevent application crashing, dont resize the height and width to 0 s, this condition checks it
+        if(newWidth==0 && newHeight==0){
+            newWidth=1366;
+            newHeight=768;
+        }
```

Comment:
What if the user's window size is not 1366x768? Maybe just don't set it to 0 at all?
```suggestion
        if (newWidth == 0 && newHeight == 0) {
            return;
        }
```